### ISSUE TITLE
Redundant screen reader output

### ISSUE THREAD

**Issue by havvamehfooz** [FreedomScientific/VFO-standards-support#933]
Summary:

For the buttons for selecting the year and month, the screen reader outputs the access key twice and the label twice.
-The double output of the access key is a JAWS bug that occurs in conjunction with aria-keyshortcuts.
-The double output of the label is caused by the transmission of the label via aria-label and aria-description.

![Image](https://github.com/user-attachments/assets/685f1f41-4036-40d7-bc8c-9002e177bf4f)

Recommendation
-Redundant screen reader output should be avoided.
-Access keys should only be transmitted as a description.
-The description should not repeat the label.
-Example: <button aria-describedby=[ID of the tooltip with the access key]>2025</button>

Additional Comments:
•	JAWS BUG - There is a double announcement of the label and the shortcut with JAWS 2026 only in VPC mode via the following native html sample:[ https://jsfiddle.net/y7v038Le/](https://jsfiddle.net/y7v038Le/)
•	The buttons have accessibility semantics according to the specification
•	The issue is not reproducible with NVDA

**Comment by JeremiahRogersVispero** [comment]
Entered bug 158652
